dll/win32/shell32/wine/pidl.h

```diff
@@ -105,8 +105,9 @@ extern "C" {
 #define PT_SHARE	0xc3
 
 #ifdef __REACTOS__
+#define PT_FOLDERTYPEMASK       0x70
 #define PT_DESKTOP_REGITEM      0x1F // => SHDID_ROOT_REGITEM
-#define PT_COMPUTER_REGITEM     0x2E // => SHDID_COMPUTER_OTHER
+#define PT_COMPUTER_REGITEM     0x2E // => SHDID_COMPUTER_?
```

Comment:
Do you mean, any of the `SHDID_COMPUTER_*` defines?

Reply:
No, I mean more research is required about the drive letter items. Specifically drive letters which failed `GetDriveType` when they were created vs actual regitems (Control panel etc in this folder). This will happen when [`SHGDFIL_DESCRIPTIONID`](https://learn.microsoft.com/en-us/windows/win32/api/shlobj_core/nf-shlobj_core-shgetdatafromidlista) gets implemented.